src/screenshot.c

```diff
@@ -178,9 +179,28 @@ screenshot_done (GObject *source,
   g_task_run_in_thread (task, send_response_in_thread_func);
 }
 
+static gboolean
+validate_type (const char *key,
+               GVariant *value,
+               GVariant *options,
+               GError **error)
+{
+  uint32_t type = g_variant_get_uint32 (value);
```

Comment:
This returns a `guint32`. Please use that.

Reply:
👍

Reply:
We've actually to some degree been migrating away from `glib` types that have standard equivalent variants, like `gint` `guint32`, `gint64` to `int`, `uint32_t` and `int64_t`, etc. They are for all intents and purposes equivalent, but there is no reason to avoid the standard ones. They also have a higher chance of resulting in looking more sensible in editors syntax highlighting.

Reply:
I noticed in the rest of the file the glib types are used so maybe keep that for consistency but you tell me what to use :shrug:

Reply:
What we've been doing, as @jadahl mentioned, is that new code should use standard types (`uint32_t`) instead of GLib types, so I think this PR should use `uint32_t` etc